blocks/subscriptions-block/components/ActionDialog/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_STYLES,
+  BUTTON_TYPES,
+  PrimaryFont,
+} from '@wpmedia/shared-styles';
+
+import './styles.scss';
+
+const ActionDialog = ({
+  actionText,
+  actionUrl,
+  reasonPromptText,
+  headlineText,
+  linkAriaLabel,
+  linkText,
+  linkPromptText,
+  linkUrl,
+  subHeadlineText,
+}) => (
+  <PrimaryFont as="div" className="xpmedia-action-dialog">
+    {reasonPromptText
+      ? (
+        <div
+          className="xpmedia-action-dialog-reason-prompt"
+          dangerouslySetInnerHTML={{ __html: reasonPromptText }}
+        />
+      )
+      : null}
+    <div className="xpmedia-action-dialog-link-prompt">
+      {linkPromptText
+        ? (
+          <span
+            className="xpmedia-action-dialog-link-prompt-pre-link"
+            dangerouslySetInnerHTML={{ __html: linkPromptText }}
+          />
+        )
+        : null}
+      <a
+        className="xpmedia-action-dialog-link-prompt-link"
+        href={linkUrl}
+        {...(linkAriaLabel ? { 'aria-label': linkAriaLabel } : {})}
```

Comment:
What's the reason for exposing `aria-label` on this link?

Reply:
I don't know.  I thought maybe it could be used.  I can remove it though if you can't think of a reason to do this.

Reply:
The first rule of aria is to not use aria. Unless the link text is unable to provide discernible text then we would need to provide a label to give assistive technologies a better label otherwise the text string of an anchor link is valid enough